### ISSUE TITLE
can't import into trakt

### ISSUE THREAD

**Issue by MrShmallow99** [jensb89/Netflix-to-Trakt-Import#12]
it gives me this, after I authorized in web browser:

```
Adding epsiode to trakt:
Traceback (most recent call last):
  File "C:\Users\XXXXX\Desktop\Netflix-to-Trakt-Import-master\netflix2trakt.py", line 222, in <module>
    time = datetime.datetime.strptime(watchedTime + " 20:15", config.CSV_DATETIME_FORMAT + " %H:%M")
  File "C:\Users\XXXXX\AppData\Local\Programs\Python\Python310\lib\_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "C:\Users\XXXXX\AppData\Local\Programs\Python\Python310\lib\_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '29/07/2022 20:15' does not match format '%d.%m.%y %H:%M'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\XXXXX\Desktop\Netflix-to-Trakt-Import-master\netflix2trakt.py", line 226, in <module>
    time = datetime.datetime.strptime(watchedTime + " 20:15", config.CSV_DATETIME_FORMAT + " %H:%M")
  File "C:\Users\XXXXX\AppData\Local\Programs\Python\Python310\lib\_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "C:\Users\XXXXX\AppData\Local\Programs\Python\Python310\lib\_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '29.07.2022 20:15' does not match format '%d.%m.%y %H:%M'
```

please help

**Comment by MrShmallow99** [comment]
Nevermind, did some digging in here.
tl;dr:
If you have a CSV file on d/m/y format, and you changed the config.py, and it still gives you error like me...
You should change the format in the config.py file from %d/%m/%y to %d/%m/%Y with capital Y.
That's it.
